src/core/border_router/routing_manager.hpp

```diff
@@ -715,6 +715,8 @@ class RoutingManager : public InstanceLocator
         void HandleRouterTimer(void);
 
     private:
+        static constexpr uint32_t kUsableOnLinkPrefixMinPreferredLifetime = 1800; // In sec.
```

Comment:
nit: Seems the [latest draft](https://www.ietf.org/archive/id/draft-ietf-snac-simple-04.html#name-suitable-on-link-prefixes) calls it `suitable` instead of `usable`.

Reply:
Changed to "favored" to be consistent with the rest of code.